web/src/components/organisms/ImgUpload.jsx

```diff
@@ -0,0 +1,18 @@
+import { FileInput, Label } from 'flowbite-react';
+import React from 'react';
+
+
+const ImgUpload = (props) => {
+    return (
+        <div>
+
+            <div>
+                <Label htmlFor="large-file-upload" value="Upload File" />
+            </div>
+            <FileInput id="large-file-upload" sizing="lg" />
+
+        </div>
+    )
```

Comment:
1. Use med size, unless needed to be big
2. Other html elements is not needed
3. Our design is accepting multiple, so let's add it

```suggestion
    return <FileInput id="large-file-upload" multiple={multiple} />
```